lib/modules/announcements.js

```diff
@@ -30,53 +31,70 @@ const recheckPostAfter = HOUR;
 
 const now = Date.now();
 
-module.go = () => {
+let post;
```

Comment:
Okay, I'll squash it into `go`. On the off-change it's slow and annoying, we can fix the UX with CSS in a hotfix or something.